### ISSUE TITLE
Improve Neural Networks and Deep Learning section

### ISSUE THREAD

**Issue by diegocasmo** [diegocasmo/ml-project#15]
### Deep learning section review (@ankurshukla03):
- unanswered questions/issues:
  - how does the network learn? (this looks like a good place to introduce supervised learning)
  - there's no mention anywhere of gradient descent/backpropagation

**Comment by ankurshukla03** [comment]
I remember Gustaf saying that there is no need of writing basic information about Neural network that's why i didn't wrote about the how the neurons are connected.

**Comment by ankurshukla03** [comment]
@diegocasmo  Should I add about backpropagation in Deep learning and it is used for learning by the network so shouldn't be written in CNN section(just a question)?

**Comment by diegocasmo** [comment]
@ankurshukla03 
IMO, back-propagation should defined in the neural networks section. See the section 'Neural Networks' in the paper 'Star-galaxy Classification Using Deep Convolutional Neural Networks' for a good example of what I think we should aim for in this section.

**Comment by ankurshukla03** [comment]
Ok cool

**Comment by diegocasmo** [comment]
@ankurshukla03 
Issues are automatically closed when the PR that has a reference to them are merged (I have updated the description of the PR to say "Closes ##" to make a reference to this issue).
